stores-api.yaml

```diff
@@ -1,95 +1,602 @@
+        schema:
+          type: string
+        example: ddf24dc6-1a2d-4391-8f34-c5c322b21c1e
+      - name: type
+        in: query
+        description: Type of image
+        required: true
+        schema:
+          type: string
+          enum:
+          - PROFILE
+          - OWNER
+        example: PROFILE
+      requestBody:
+        content:
+            # a binary file of type png or jpeg
+          'image/jpeg':
+            schema:
+              type: string
+              format: binary
+          'image/png':
+            schema:
+              type: string
+              format: binary        
+      responses:
+        200:
+          description: ok
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Store'
+  /stores/{id}/products:
+    post:
+      tags:
+      - Stores API
+      - Products API
+      summary: Create a new product for the store
+      security: 
+      - basicAuth: []
+      parameters:
+      - name: id
+        in: path
+        description: id
+        required: true
+        schema:
+          type: string
+        example: ddf24dc6-1a2d-4391-8f34-c5c322b21c1e
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ProductDraft'
+      responses:
+        200:
+          description: ok
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Product'
+        500:
+          description: An unexpected error occurred
           content: {}
 components:
   schemas:
-    StoreItem:
-      required:
-      - id
-      - name
+    Address:
+      title: Address
       type: object
       properties:
+        city:
+          type: string
+        country:
+          type: string
+          description: Alpha-2
         id:
-          $ref: '#/components/schemas/Id'
+          type: string
+        postalCode:
+          type: string
+        region:
+          type: string
+        state:
+          type: string
+        streetName:
+          type: string
+        streetNumber:
+          type: string
+    Attribute:
+      title: Attribute
+      type: object
+      properties:
         name:
           type: string
-          example: grocery store          
-    Id:
-      type: string
-      format: uuid
-      example: d290f1ee-6c54-4b01-90e6-d701748f0851
+    GeoJSON:
+      title: GeoJSON
+      type: object
+      discriminator:
+        propertyName: type
+    Image:
+      title: Image
+      type: object
+      properties:
+        dimensions:
+          $ref: '#/components/schemas/ImageDimensions'
+        label:
+          type: string
+        url:
+          type: string
+    ImageDimensions:
+      title: ImageDimensions
+      type: object
+      properties:
+        h:
+          type: integer
+          format: int32
+        w:
+          type: integer
+          format: int32
```

Comment:
What unit are we expecting? Probably pixels?

Reply:
Oh, and why do we need image dimensions?

Reply:
It's from the original CT API. I'd just leave it in